openlibrary/plugins/openlibrary/js/dialog.js

```diff
@@ -0,0 +1,106 @@
+import initTabs from './tabs';
+import { placeholder } from './jquery-ui';
+
+/**
+ * @param {string} selector of element that will open dialog
+ * @param {string} href of element that will be the contents of the dialog
+ */
+function setupDialog(selector, href) {
```

Comment:
Can we rename `href` to `dialogSelector`? `href` makes it seem like it could be an arbitrary link.